common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -7026,15 +7033,15 @@ public static void setHiveSiteLocation(URL location) {
   }
 
   public static void setHivemetastoreSiteUrl(URL location) {
-    hivemetastoreSiteUrl = location;
+    metastoreSiteUrl = location;
   }
 
   public static URL getHiveSiteLocation() {
     return hiveSiteURL;
   }
 
   public static URL getMetastoreSiteLocation() {
-    return hivemetastoreSiteUrl;
+    return metastoreSiteUrl;
   }
 
   public static URL getHiveServer2SiteLocation() {
```

Comment:
is it the same story as with `hivemetastore-site.xml`? is it even used? i don't see we use it in the `initialize`